apps/rewards/app/components/Content/Overview.js

```diff
@@ -211,7 +222,8 @@ const tableType = [
 const Overview = ({ tokens, rewards, convertRates, claims, newReward, openDetails }) => {
   const rewardsEmpty = rewards.length === 0
 
-  console.log('reward props: ', rewards)
+  // For testing purposes uncomment the following line:
+  // console.log('reward props: ', rewards)
```

Comment:
I doubt this comment is useful. We probably all know how to add console logging when we need it! 😄

Reply:
True :)